content/docs/5.1/concepts/explore-tikv-features/replication-and-rebalancing.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Replication and rebalancing
+description: Replication and rebalancing
+menu:
+    "5.1":
+        parent: Features
+        weight: 1
+---
+
+This page walks you through a simple demonstration of how TiKV replicates, distributes, and rebalances data. Starting with a 3-node local cluster, you'll write some data via [go-ycsb](https://github.com/pingcap/go-ycsb) and verify that it replicates in triplicate by default. You'll then add 2 more nodes and watch how TiKV automatically rebalances replicas to efficiently use all available capacity. **NOTE: Do not apply this on production.**
+
+
+## Before you begin
+
+Make sure you have install [TiUP](https://github.com/pingcap/tiup) version **v1.5.2** or above.
+
+
+## Step 1. Start a 3-node cluster
+
+1. Use the `tiup-playground` to start a 3-node local cluster.
+    ```sh
+    tiup playground v5.0.2 --mode tikv-slim --kv 3
+    ```
+
+    The Command above will output the following content:
+    ```txt
+    ...
+    Starting component `playground`: /home/pingcap/.tiup/components/playground/v1.5.0/tiup-playground --mode tikv-slim --kv 3
+    Using the version v5.0.2 for version constraint "".
+
+    If you'd like to use a TiDB version other than v5.0.2, cancel and retry with the following arguments:
+        Specify version manually:   tiup playground <version>
+        Specify version range:      tiup playground ^5
+        The nightly version:        tiup playground nightly
+
+    Playground Bootstrapping...
+    Start pd instance
+    Start tikv instance
+    Start tikv instance
+    Start tikv instance
+    PD client endpoints: [127.0.0.1:2379]
+    To view the Prometheus: http://127.0.0.1:33703
+    To view the Grafana: http://127.0.0.1:3000
+    ```
+
+## Step 2. Write data
+
+On another terminal session, we will use [go-ycsb](https://github.com/pingcap/go-ycsb) to launch a workload.
+
+1. Clone the `go-ycsb` from GitHub.
+    ```sh
+    git clone https://github.com/pingcap/go-ycsb.git
+    ```
+2. Then build the application from source.
+    ```sh
+    make
+    ```
+3. load a small workload using `go-ycsb`
+    ```
+    # By default, this workload will insert 1000 records into TiKV
+    ./bin/go-ycsb load tikv -P workloads/workloada -p tikv.pd="127.0.0.1:2379" -p tikv.type="raw" 
+    ```
```

Comment:
```suggestion
3. load a small workload using `go-ycsb`

    ```bash
    # By default, this workload will insert 1000 records into TiKV
    ./bin/go-ycsb load tikv -P workloads/workloada -p tikv.pd="127.0.0.1:2379" -p tikv.type="raw" 
    ```
```

Reply:
Done.